src/sklearn_evaluation/plot/clustering.py

```diff
@@ -78,28 +87,47 @@ def elbow_curve(X, clf, n_clusters=None, n_jobs=1, show_cluster_time=True, ax=No
     ax: matplotlib Axes
         Axes containing the plot
 
+    Notes
+    -----
+    .. deprecated:: 0.8.7
+        'n_clusters' renamed to 'range_n_clusters' and will be removed in version 0.8.9
```

Comment:
this shouldn't be 0.8.9, it should be 0.9. we're following semantic versioning and breaking API changes only happen in major releases